Makefile

```diff
@@ -21,7 +21,11 @@ install-base: install-base-pip-packages install-nltk-models
 install: install-base-pip-packages install-dev install-nltk-models install-test install-huggingface install-all-docs
 
 .PHONY: install-ci
-install-ci: install-base-pip-packages install-nltk-models install-huggingface install-all-docs install-test
+install-ci: install-base-pip-packages install-nltk-models install-huggingface install-all-docs install-test install-local-inference-branch
+
+.PHONY: install-local-inference-branch
+install-local-inference-branch:
+	git clone -b  yuming/remove_ocr_code --single-branch https://github.com/Unstructured-IO/unstructured-inference.git && cd unstructured-inference && pip install -e . && cd ../
```

Comment:
this change is for review ci tests only, since all the tests need to use the updated inference repo with ocr related code removed

Reply:
also the same purpose for all changes related to CI files, just to make sure CI use the inference refactor branch. will be removed after the inference refactor branch is merged and publish a new inference version...